client/verta/tests/unit_tests/registry/test_model_dependencies.py

```diff
@@ -52,6 +53,16 @@ def test_modules_in_function_body_return_line(dependency_testing_model) -> None:
     assert extracted_modules == expected_modules
 
 
+def test_modules_in_function_body_as_class_instance(dependency_testing_model) -> None:
+    """ Verify that modules introduced only via class instance are extracted
+    as expected.
```

Comment:
What does the "instance" in "class instance" mean exactly? Isn't it just a "class", not an instance of one?

Reply:
The means by which this module is introduced is the declaration of an instance of a class provided by a 3rd-party module.  Meaning `DataFrame(some_data)` declares an instance of the DataFrame class, vs. a type annotation that may simply use the class itself `df: DataFrame = make_a_df()`.  The distinction matters because each case is handled in a slightly different manner.